torch/_dynamo/utils.py

```diff
@@ -814,6 +815,7 @@ class CompilationMetrics:
 
 @dataclasses.dataclass
 class BwdCompilationMetrics:
+    is_forward: bool = dataclasses.field(default=False, init=False)
```

Comment:
By the way it's so weird to me that you put this at the top of the dataclass, it feels very BC breaking to do it that way if people are passing in arguments positionally lol

Reply:
like if tests pass I guess it's fine it's just weird lol

Reply:
@ezyang, yeah I normally wouldn't have, but I tried to make this field essentially a static property of the class -- not provided at initialization. So somehow it made more sense to me to stick it at the top. The next time someone appends a new field to this dataclass, I'm hoping they won't be confused about why is_forward is not passed.